zerver/lib/actions.py

```diff
@@ -1759,15 +1759,14 @@ def do_change_password(user_profile, password, commit=True,
                                  modified_user=user_profile, event_type='user_change_password',
                                  event_time=event_time)
 
-def do_change_full_name(user_profile, full_name, log=True):
-    # type: (UserProfile, Text, bool) -> None
+def do_change_full_name(user_profile, full_name, acting_user):
```

Comment:
hm, there exist paths in the UI that allow an admin to change a user's name (outside of the management commands), but this commit makes it seem like the acting_user is always going to be either the user_profile or None.